lib/theme.js

```diff
@@ -1,7 +1,15 @@
-const colors = require('colors');
+/* eslint-disable no-console */
+const chalk = require('chalk');
 
-colors.setTheme({
-  info: 'green',
-  warn: 'yellow',
-  error: 'red',
-});
+exports.print = function print(msg, level) {
+  if (level === 'info') console.log(chalk.green(msg));
+  else if (level === 'warn') console.log(chalk.yellow(msg));
+  else if (level === 'error') console.log(chalk.red(msg));
+  else if (level === 'data') console.log(chalk.magenta(msg));
+  else console.log(msg);
+}
+
+exports.ERROR = 'error';
+exports.INFO = 'info';
+exports.WARN = 'warn';
+exports.DATA = 'data';
```

Comment:
it's a bit annoying to have to import individual log levels. maybe have a Map/Object that acts as a levels enum

```
exports.Levels = Object.freeze({
  ERROR: 'error',
  INFO: 'info',
  WARN: 'warn',
  DATA: 'data',
});
```

Reply:
Learned about Object.freeze today